Driverless AI/disparate-impact-analysis-tutorial/disparate-impact-analysis-tutorial.md

```diff
@@ -0,0 +1,458 @@
+
+
+## Task 3: Confusion Matrix
+
+
+After the model is interpreted, you will be taken to the "MLI: Regression and Classification Explanations" page. The *DAI Model* tab should already be selected for you thereafter, click on *Disparate Impact Analysis* located at the bottom left corner of the page. The following page should appear: 
+
+![dia](assets/dia.jpg)
+
+Before we continue with the experiment, we need to cover one more  important concept: Confusion Matrices. 
+
+It is effortless to assume that the goal is accuracy when using machine learning: the percentage of your predictions being correct. While accuracy can be a useful metric of success, it is often dubious. Let’s build on a very relevant industry example: 
+
+Fraud (Anomaly Detection): Let’s assume we are dealing with a 100,000-row data set where we know there is some small amount of fraud; let’s say 10. If accuracy is our benchmark, then your model will predict “Not-Fraud” every time, and the accuracy will be 99.99%, but you have failed to identify any instances of fraud. These cases focus on what in a confusion matrix is defined as True Positives (TP).
+
+![confusion-matrices-1](assets/confusion-matrices-1.jpg)
+
+In the case of identifying fraud, you would almost always prefer a prediction table like this, to make sure you can correctly identify fraud instances as they occur:
+
+![confusion-matrices-2](assets/confusion-matrices-2.jpg)
+
+Now every case is different, and often in business environments, there exist certain cost functions associated with false negatives and false positives, so it is essential to be aware that every case has many considerations. We want to provide a few of the key metrics associated with confusion matrices that come up in the industry, depending on the problem you are trying to solve.
+
+Sensitivity, Recall, Hit Rate, True Positive Rate:
+ - True Positive Rate = True Positive/ Condition Positive
+
+Specificity, Selectivity, True Negative Rate:
+
+- True Negative Rate = True Negative/Condition Negative
+
+Precision, Positive Predictive Value:
+
+- Precision =  True Positive/Predicted Condition Positive
+
+
+
+With this context in mind, let’s move forward and dive into the experiment!
+
+1. Observe the Global Confusion Matrix for our experiment:
+
+![dia-2](assets/dia-2.jpg)
+
+2. With the above in mind, you can look at the Global Confison Metrix of our experiment and identify and calculate the *True Positive Rate* and *True Negative Rate*. In the following task we will see how *True Positive Rate* and *True Negative Rate* can be effective. 
+
+*Note:* In the next tutorial (COMING SOON), we will be exploring how we can use  *True Positive Rate* and *True Negative Rate* to determine the fairness of an AI model. Therefore, let us explore the two new MLI tools: 
+1. Disparate Impact Analysis Tool 
+
+ 2. Sensitivity/What-If Analysis Tool 
+
+
```

Comment:
Do you have any Deeper Dive or Resouces you can add for this section? Perhaps we can add a link  to the DAI Tutorial 2, concepts section that goes over confusion matrices in more depth.

Reply:
@Castro38 the tutorial goes over confusion matrices